frontend/src/components/projects/NewProject.tsx

```diff
@@ -11,13 +11,16 @@ const NewProject: FunctionComponent = () => {
   const [projectForm, setProjectForm] = useState(false);
 
   useEffect(() => {
+    let isMounted = true;
```

Comment:
would be good to look at whether this check is still needed.

Reply:
I removed it and got the same error so I'm thinking we should leave it in